webapp/src/AppComponents/DeleteWorkbookDialog.tsx

```diff
@@ -55,10 +63,25 @@ export const DeleteWorkbookDialog = forwardRef<
       </ContentWrapper>
     </DialogWrapper>
   );
-});
+}
 
 DeleteWorkbookDialog.displayName = "DeleteWorkbookDialog";
 
+// some colors taken from the IronCalc palette
```

Comment:
We can't use the palette. This is part of the app we are building that is built on the top of the IronCalc widget.

Maybe we could export those colors somehow.